triedb/firewood/database.go

```diff
@@ -99,17 +101,17 @@ func New(config *Config) *Database {
 
 	fwConfig, err := validatePath(config)
 	if err != nil {
-		log.Crit("firewood: error validating config", "error", err)
+		fmt.Println("firewood: error validating config", "error", err)
 	}
 
 	fw, err := ffi.New(config.FilePath, fwConfig)
 	if err != nil {
-		log.Crit("firewood: error creating firewood database", "error", err)
+		fmt.Println("firewood: error creating firewood database", "error", err)
 	}
 
 	currentRoot, err := fw.Root()
 	if err != nil {
-		log.Crit("firewood: error getting current root", "error", err)
+		fmt.Println("firewood: error getting current root", "error", err)
```

Comment:
we should not use printing for 2 reasons:
* this does not return in case of an error (despite log.Crit where it exits the program)
* rpcchainvms are not compatible with fmt.Println so this won't work with subnet-evm and we really should be using a logger for this.